src/core/tools/common/simple_logger.cpp

```diff
@@ -27,11 +27,32 @@
 #include "simple_logger.h"
 #include <sstream>
 #include <dsn/utility/filesystem.h>
+#include <dsn/utility/flags.h>
 #include <dsn/utility/time_utils.h>
 
 namespace dsn {
 namespace tools {
 
+DSN_DEFINE_bool("tools.simple_logger", fast_flush, false, "whether to flush immediately");
+
+DSN_DEFINE_bool("tools.simple_logger",
+                short_header,
```

Comment:
if the value of one config format isn't right, what happen?

Reply:
The behavior is exactly the same as `dsn_config_get_value_int64` like formerly it does.